sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/json/JsonBenchmark.scala

```diff
@@ -508,6 +548,9 @@ object JsonBenchmark extends SqlBasedBenchmark {
       jsonInDS(50 * 1000 * 1000, numIters)
       jsonInFile(50 * 1000 * 1000, numIters)
       datetimeBenchmark(rowsNum = 10 * 1000 * 1000, numIters)
+      // Benchmark pushdown filters that refer to top-level columns.
+      // TODO: Add benchmarks for filters with nested column attributes.
```

Comment:
It's better we have an IDed TODO like `TODO(SPARK-XXX)` with the explicit SPARK JIRA ID.

Reply:
@dbtsai @dongjoon-hyun There is an umbrella ticket for nested fields https://issues.apache.org/jira/browse/SPARK-25556 but it mentions only Parquet and it has been resolved already. Should I add a sub-task there and re-open it, or open separate JIRA?

Reply:
I created the sub-task https://issues.apache.org/jira/browse/SPARK-32325